src/main/webapp/app/shared/user-settings/user-settings-prototype/user-settings-prototype.component.html

```diff
@@ -0,0 +1,41 @@
+<!--
+       Exemplary HTML template for UserSettings
+       Every settings page will be somewhat unique, especially its optionCore (different properties and methods)
+       This nested behavior could not be realized via polymorphism due to different types and strong bidirectional dependencies
+       Therefore just copy this template for new settings and modify it. The minimal adjustment needed for the template to work is to implement the
+       option core with its properties and functionalities under <! --OptionCore-- >
+       Look at notification-settings.component.html for a good example
+
+<jhi-alert></jhi-alert>
+<div class="d-flex justify-content-between">
+    <h1>
+        <! --UserSettingsCategory-- >
+        {{ 'artemisApp.userSettings.categories.' + (userSettingsCategory | lowerCamelCase) | artemisTranslate }}
+    </h1>
+    <button type="button" class="btn btn-primary" *ngIf="optionsChanged" id="apply-changes-button" (click)="saveOptions()">
+        <fa-icon [icon]="'save'"></fa-icon>
+        {{ 'artemisApp.userSettings.saveChanges' | artemisTranslate }}
+    </button>
+</div>
+<div class="list-group d-block" *ngIf="userSettings">
+    <div *ngFor="let optionGroup of userSettings.groups">
+        <div *jhiHasAnyAuthority="[optionGroup.restrictionLevel]" class="list-group-item">
+            <h3>
+                <! --GroupName-- >
+                {{ 'artemisApp.userSettings.optionGroupNames.' + (optionGroup.name | lowerCamelCase) | artemisTranslate }}
+            </h3>
+            <div *ngFor="let option of optionGroup.options">
+                <dt>
+                    <! --OptionName-- >
+                    {{ 'artemisApp.userSettings.optionNames.' + (option.name | lowerCamelCase) | artemisTranslate }}
+                </dt>
+                <span>
+                    <! --OptionDescription-- >
+                    {{ 'artemisApp.userSettings.optionDescriptions.' + (option.description | lowerCamelCase) | artemisTranslate }}
+                </span>
+                <! --Put Settings specific core properties & functions here-- >
+            </div>
+        </div>
+    </div>
+</div>
+-->
```

Comment:
seems like everything is commented out, can we remove it?

Reply:
To support reusability I commented this file out for future (different) settings, so this file can be copied and reused. (Why not just keep the notification-settings template then? -> This notification template may change and differ strongly from other future settings therefore I wanted to provide the bare minimum to be comfortably reused and that supports the new settings data structure hierarchy by dynamic angular template rendering (i.e. ngFor loops)
The problem here also is that the user-settings-prototype.component needs a template. (I tried using "" but this does not work)